File: 453.MinimumMovesToEqualArrayElements/move.cc
class Solution {
public:
    int minMoves(vector<int>& nums) {
        int sum = 0, minVal = nums[0];
        for (int i = 0; i < int(nums.size()); i++) {
            sum += nums[i];
            minVal = min(minVal, nums[i]);
        }
        return sum - minVal * nums.size();
    }
};

File: 453.MinimumMovesToEqualArrayElements/Move.java
class Solution {
    public int minMoves(int[] nums) {
        int sum = 0, min = nums[0];
        for (int i = 0; i < nums.length; i++) {
            sum += nums[i];
            min = Math.min(min, nums[i]);
        }
        return sum - min * nums.length;
    }
}
